flags.ts

```diff
@@ -25,4 +26,20 @@ export const DEFAULT_FLAGS = [
   '--mute-audio',
   // Skip first run wizards
   '--no-first-run',
+
+  // Disable timers being throttled in background pages/tabs
+  '--disable-background-timer-throttling',
+  // Disables client-side phishing detection. Likely redundant due to the safebrowsing disable
+  '--disable-client-side-phishing-detection',
+  // Disable popup blocking
+  '--disable-popup-blocking',
+  // Reloading a page that came from a POST normally prompts the user.
+  '--disable-prompt-on-repost',
+  // Disable the a few things considered not appropriate for automation, e.g. infobar animations, password saving UI
+  //   https://docs.google.com/a/google.com/document/d/1JYj9K61UyxIYavR8_HATYIglR9T_rDwAtLLsD3fbDQg/preview
```

Comment:
this link doesn't actually reference that flag, do we have something that explains what it does? or is `--enable-automation` the same as `--enable-chromedriver`?

Reply:
Yah in short that design doc turned into --enable-automation during code review but the doc wasn't updated. 
CL here: https://codereview.chromium.org/2564973002 where pavel recommended the flag be generalized. :)

> My only comment is to call it in a generic manner: --enable-automation, AutomationInfobar, etc. It is likely that Lighthouse would also like to use it the same way ChromeDriver does.

it adds this infobar: 

![image](https://user-images.githubusercontent.com/39191/30349667-92a7a086-97c8-11e7-86b2-1365e3d407e3.png)

and disables dev mode extension bubbles (?), and doesn't show some other info bars
and it means the default browser check prompt isn't shown
and it avoids showing these 3 infobars: ShowBadFlagsPrompt, GoogleApiKeysInfoBarDelegate, ObsoleteSystemInfoBarDelegate